modules/lss-shareAlliancePost/ShareAlliancePost.js

```diff
@@ -582,13 +582,9 @@
                 const missionID = missionlink
                     .replace(/\?.*$/, '')
                     .match(/\d*$/)[0];
-                const langCode = I18n.currentLocale();
-                fetch(
-                    `https://proxy.lss-manager.de/api/missions.php?lang=${langCode}&mission=${missionID}`,
-                    { cache: 'no-cache' }
-                )
-                    .then(res => res.json())
-                    .then(data => {
+                let requirements = JSON.parse(localStorage.aMissions).value;
+                if(requirements.filter(e => e.id == parseInt(missionID))[0] == undefined) await $.getJSON('/einsaetze.json').done(data => {localStorage.setItem('aMissions', JSON.stringify({lastUpdate: new Date().getTime(), value: data})); requirements = data;})
```

Comment:
The function must be async to get this await working

Reply:
Sorry, locaI I changed it, but forget to change it here too.